Godeps/_workspace/src/github.com/Clever/gearadmin/.drone.yml

```diff
@@ -0,0 +1,13 @@
+image: go1.2
```

Comment:
I may be missing something, but why the version difference here and in the godeps.json ? does this image actually have go1.3?